package.json

```diff
@@ -67,5 +67,6 @@
     "simulant": "^0.2.2",
     "stringify": "^5.1.0",
     "vinyl-source-stream": "^1.1.0"
-  }
+  },
+  "dependencies": {}
```

Comment:
just curious, was there a problem not having a `"dependencies"` key in package.json?

Reply:
I don't believe so, I never added it so an `npm i` must have